src/tree.types.ts

```diff
@@ -25,7 +25,35 @@ export interface TreeModel {
   _foldingType?: FoldingType;
 }
 
+export interface CssClasses {
+  /* The class or classes marking a expanded state */
+  expanded?: string;
+
+  /* The class or classes marking a collapsed state */
+  collapsed?: string;
+
+  /* The class or classes marking an inactive state - node is a leaf */
```

Comment:
You're right about inactive, at the begin cssClasses and templates was one object and it wasn't possible to name two different properties the same way, but during development I descided to split them. So I renamed inactive to leaf.